user-interface/src/lib/components/uswds/Checkbox.test.tsx

```diff
@@ -1,7 +1,7 @@
 import { render, screen, fireEvent } from '@testing-library/react';
 import React from 'react';
 import { BrowserRouter } from 'react-router-dom';
-import Checkbox, { CheckboxProps, CheckboxRef } from './Checkbox';
+import Checkbox, { CheckboxProps, CheckboxRef, CheckBoxState } from './Checkbox';
```

Comment:
`CheckBoxState` should be `CheckboxState` to match the other spellings.

Reply:
Updated 24 occurrances.